acnportal/acnsim/tests/test_interface.py

```diff
@@ -39,9 +43,10 @@ def test_is_feasible_unequal_schedules(self):
                 {'PS-001' : [1, 2], 'PS-002' : [3, 4, 5], 'PS-003' : [4, 5]})
 
     def test_is_feasible(self):
-        self.network.station_ids = ['PS-002', 'PS-001', 'PS-003']
+        self.simulator.network = create_autospec(ChargingNetwork)
+        self.simulator.network.station_ids = ['PS-002', 'PS-001', 'PS-003']
         _ = self.interface.is_feasible({'PS-001' : [1, 2], 'PS-002' : [4, 5]})
-        network_is_feasible_args = self.network.is_feasible.call_args
+        network_is_feasible_args = self.interface._simulator.network.is_feasible.call_args
```

Comment:
Why use the network through the interface here instead of just using self.network?

Reply:
I had set `self.simulator.network` to a new mocked network so I could look at `is_feasible` call args. I pushed a change where I just mock `self.network.is_feasible` directly, which I think is better.